packages/plugin/vite/src/VitePlugin.ts

```diff
@@ -0,0 +1,160 @@
+import fs from 'node:fs';
+import http from 'node:http';
+import path from 'node:path';
+
+import { namedHookWithTaskFn, PluginBase } from '@electron-forge/plugin-base';
+import { ForgeMultiHookMap, StartResult } from '@electron-forge/shared-types';
+import debug from 'debug';
+// eslint-disable-next-line node/no-extraneous-import
+import { RollupWatcher } from 'rollup';
+import { default as vite } from 'vite';
+
+import { VitePluginConfig } from './Config';
+import ViteConfigGenerator from './ViteConfig';
+
+const d = debug('electron-forge:plugin:vite');
+
+export default class VitePlugin extends PluginBase<VitePluginConfig> {
+  name = 'vite';
+
+  private isProd = false;
+
+  // The root of the Electron app
+  private projectDir!: string;
+
+  // Where the Vite output is generated. Usually `${projectDir}/.vite`
+  private baseDir!: string;
+
+  private _configGenerator!: ViteConfigGenerator;
+
+  private watchers: RollupWatcher[] = [];
+
+  private servers: http.Server[] = [];
+
+  init = (dir: string): void => {
```

Comment:
```suggestion
  public init(dir: string): void => {
```

Reply:
Okay. That seems to be the case.

https://github.com/electron/forge/blob/0720e070480ad777ef6a94425a4ceaf43fc12fad/packages/plugin/compile/src/CompilePlugin.ts#L14-L20